src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -53,6 +62,25 @@ const generateLabelSpanContents = (labelValue, key,  spanValue, isLoading) => {
   );
 };
 
+// content generated by function
+export function DynamicRowGeneration = (rowElementMaxCount, labelArray) => {
+  // todo: QUESTION - should I use generateLabelSpanContents before or after chunking?
+  // todo: QUESTION - should I apply use this function for the Applicant Information Section as well?
```

Comment:
You don't need to apply it there, the contents on Applicant Information are always visible, so a standard templating approach works there.